### ISSUE TITLE
Login - Forgot Password option

### ISSUE THREAD

**Issue by halender** [opencb/jsorolla#392]
## Description 

We need to implement a password reset screen, so the user can easily reset the password of his account using the web interface.

To access this reset password screen, we will add a link in the login screen:

![image](https://user-images.githubusercontent.com/35806821/151988094-3fed3358-8c8e-4bfb-b408-2ac076427739.png)

In the reset password screen, the user will be prompted for his user ID, so we can make a request to the reset password endpoint and send an email to the user with the steps to reset the password.

## Technical solution

OpenCGA endpoint: [TODO]

## Tasks

- [x] Implement `user-password-reset` webcomponent.
- [x] Register this new screen in IVA.
- [x] Add link in login screen to redirect to the reset password screen.
- ~~Call OpenCGA endpoint to reset user password.  (blocked: waiting for opencga endpoint creation)~~

**Comment by jmjuanes** [comment]
First implementation of the reset password screen: 

![image](https://user-images.githubusercontent.com/5751201/152389234-713c349b-f91a-4e93-8dd0-0e0a5127df31.png)

**Comment by jmjuanes** [comment]
Refactored user login component in commit 87c8266 and added link to reset-password view: 

![image](https://user-images.githubusercontent.com/5751201/153203947-3358289c-bbb4-4f42-8ead-3277ab74bc93.png)

**Comment by jmjuanes** [comment]
Moved to **Waiting for feedback**. In order to finish this task we need the Reset password endpoint deployed in OpenCGA.

**Comment by halender** [comment]
This endpoint is the Autogenerated Command-Line Branch. @juanfeSanahuja, please notify us once this branch is merged in dev and deployed in the test environment.

**Comment by jmjuanes** [comment]
Finally, the endpoint for allowing a user to reset its password has been delayed to future releases. For that reason, **the reset password screen has been disabled** and only the login redesign is visible as part of the development made on this ticket.

![image](https://user-images.githubusercontent.com/5751201/158867209-7e3dc264-2c85-4a20-aac3-5040221dcf47.png)

**Comment by halender** [comment]
Thank you @jmjuanes. 
@pamag. Could you test the disabled password screen and open a new ticket for this in 2.3?

**Comment by pamag** [comment]
FIXED and new ticket in clickup for 2.3 for password reset
